### ISSUE TITLE
[Bug]: 'Checks & Saves' page gives a broken example of a passive Wisdom check

### ISSUE THREAD

**Issue by CS-Birb** [foundryvtt/foundryvtt-premium-content#489]
### What happened?

Unclear whether this is a problem with the new Passive checks enrichers, or if the examples given are incorrect - so I'll make a similar bug report in the DnD5e system repo.

In the 'Checks & Saves' page of the 'Foundry and Adventure Advice' journal entry, the 'With DC' row gives an example of a passive check against the wisom ability `[[/check wisdom dc=12 passive=true]]`, while the lower rows each perform a check against the Perception skill. This example just shows a loading spinner and silently logs an error to the console.
The passive check handled at [tooltips.mjs#L122](https://github.com/foundryvtt/dnd5e/blob/3.0.x/module/tooltips.mjs#L122) only handles skills rather than direct ability checks.

Similarly, as the enricher refers directly to the actor's skills swapping out the ability, as in the 'Using Non-Standard Ability' example, does not have the suggested impact. `[[/check perception ability=int passive=true dc=15]]` still evaluates against the wisdom modifier rather than intelligence modifier.

### What package are you reporting this for?

Phandelver and Below: The Shattered Obelisk

### Version

1.0.0

### Relevant log output

```shell
tooltips.mjs:124 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'label')
    at Tooltips5e._onHoverPassive (tooltips.mjs:124:78)
    at Tooltips5e._onTooltipActivate (tooltips.mjs:81:19)
    at Tooltips5e._onMutation (tooltips.mjs:52:26)
```

**Comment by CS-Birb** [comment]
Both bugged scenarios have been accepted as bugs in the DnD5e system, and flagged for the 3.0.2 milestone.

Likely no change will be needed on the PBSO end.

**Comment by CS-Birb** [comment]
Closing as https://github.com/foundryvtt/dnd5e/issues/2883 is now closed and rectifies the issues.
Will be fixed in 3.0.2 release.
